### ISSUE TITLE
Scrollable projections

### ISSUE THREAD

**Issue by simonbyrne** [MakieOrg/GeoMakie.jl#270]
Firstly, I am really impressed by how much this package has improved! Kudos to all the hard work.

What would it take to make the projections themselves scrollable, i.e. scrolling will change the projection itself, rather than the panning/zooming on the projected image. I realize this is quite ambitious, but it would be really cool to e.g. scroll around an orthographic projection.

**Comment by SimonDanisch** [comment]
Making the projection an observable should already be supported ;)
You'd just need to hook that up to the mouse events to change the projection appropriately.
If creating each projection is fast, I think this should also have reasonable performance.

**Comment by asinghvi17** [comment]
Yes this would be pretty straightforward - there's an animation in the examples folder which shows how you would "scroll" a projection.  From there it's pretty much smooth sailing.

I have wanted to do this for a while so might push up an example in the next few days!
